lightwood/mixers/helpers/selfaware.py

```diff
@@ -0,0 +1,61 @@
+import torch
+import math
+
+from lightwood.config.config import CONFIG
+from lightwood.helpers.device import get_devices
+
+
+# Todo: conform to style guidelines
+class SelfAware(torch.nn.Module):
+    def __init__(self, input_size, output_size, nr_outputs):
+        super(SelfAware, self).__init__()
+
+        self.input_size = input_size
+        self.output_size = output_size
+        self.nr_outputs = nr_outputs
+
+        awareness_layers = []
+        awareness_net_shape = [(self.input_size + self.output_size),
+                               max([int((self.input_size + self.output_size) * 1.5), 300]),
+                               self.nr_outputs]
+
+        for ind in range(len(awareness_net_shape) - 1):
+            rectifier = torch.nn.SELU
+            awareness_layers.append(torch.nn.Linear(awareness_net_shape[ind], awareness_net_shape[ind + 1]))
+            if ind < len(awareness_net_shape) - 2:
+                awareness_layers.append(rectifier())
+
+        self.net = torch.nn.Sequential(*awareness_layers)
+
+        for layer in self.net:
```

Comment:
Move to a helper that is shared

Reply:
Ideally keep them separate actually, since we might want to differentiate between the deafultNet and the selfaware net, we can move boilerplate to helper functions as we go.